src/main/scala/keystoneml/workflow/AutoCacheRule.scala

```diff
@@ -50,7 +50,7 @@ class AutoCacheRule(cachingMode: CachingStrategy) extends Rule with Logging {
 
   /**
    * Get an estimate for how many times the output of each node will be accessed, assuming
-   * the given set of nodes have their outputs cached.
+   * the given set of keystoneml.nodes have their outputs cached.
```

Comment:
This file has some too, but the occurrences here are mostly okay with `keystoneml.nodes` in the comments